.github/ISSUE_TEMPLATE/1.bug_report.yaml

```diff
@@ -0,0 +1,88 @@
+name: Bug report
+description: Create a report to help us improve Amplify UI
+
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thank you for taking the time to fill out this bug report! Please try to provide as much information as you can.
+  
+  - type: checkboxes
+    attributes:
+      label: |
+        Before creating a new issue, please confirm:
+      options:
+        - label: I have [searched for duplicate or closed issues](https://github.com/aws-amplify/amplify-ui/issues?q=is%3Aissue+) and [discussions](https://github.com/aws-amplify/amplify-ui/discussions).
+          required: true
+        - label: I have read the guide for [submitting bug reports](https://github.com/aws-amplify/amplify-ui/blob/main/CONTRIBUTING.md#bug-reports).
+          required: true
+
+  - type: markdown
+    attributes:
+      value: |
+        ## Environment
+  - type: dropdown
+    attributes:
+      label: On which framework/platform are you having an issue? 
+      description: Please select one or more platform!
+      multiple: true
+      options:
+        - React
+        - Angular
+        - Vue
+        - React Native
+        - Flutter
+        - iOS
+        - Android
+        - Other
```

Comment:
Should we add an option for vanilla web?

Reply:
Customer will be able to multi select on this option, if its worth having Vanilla Web option, I will add it. @ericclemmons  thoughts?

Reply:
`Other` may be sufficient for now. "Vanilla" can end up being webpack, parcel, Vite, CDN/Unpkg, etc., so we would need additional information on their _bundling_ to diagnose.